server/client_server.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package main
+
+import (
+	"net/http"
+	"strconv"
+
+	jira "github.com/andygrunwald/go-jira"
+	"github.com/pkg/errors"
+)
+
+type jiraServerClient struct {
+	JiraClient
+}
+
+func newServerClient(jiraClient *jira.Client) Client {
+	return &jiraServerClient{
+		JiraClient: JiraClient{
+			Jira: jiraClient,
+		},
+	}
+}
+
+// GetCreateMeta returns the metadata needed to implement the UI and validation of
+// creating new Jira issues.
+func (client jiraServerClient) GetCreateMeta(options *jira.GetQueryOptions) (*jira.CreateMetaInfo, error) {
+	cimd, resp, err := client.Jira.Issue.GetCreateMetaWithOptions(options)
+	if err != nil {
+		if resp == nil {
+			return nil, err
+		}
+		resp.Body.Close()
+		if resp.StatusCode == http.StatusForbidden || resp.StatusCode == http.StatusUnauthorized {
+			err = errors.New("not authorized to create issues")
+		}
+		return nil, RESTError{err, resp.StatusCode}
+	}
+	return cimd, nil
+}
+
+// SearchUsersAssignableToIssue finds all users that can be assigned to an issue.
+func (client JiraClient) SearchUsersAssignableToIssue(issueKey, query string, maxResults int) ([]jira.User, error) {
```

Comment:
Why is this JiraClient?

Reply:
Cut&paste error - curiously it just worked because the Server version overrode it, good catch, TY!